### DISCUSSION TITLE
Road to OKD 4.11

### DISCUSSION THREAD

**Discussion by vrutkovs** [okd-project/okd#1282] [Releases]
* [x] Branch okd-machine-os in release-4.11

* [x] Create release-controller configuration

* [x] Update okd-machine-os release-4.11 branch to use stable fedora-coreos-config

* [x] Setup upgrade readiness jobs

* [x] Monitor [upgrade job](https://prow.ci.openshift.org/job-history/gs/origin-ci-test/logs/periodic-ci-openshift-release-master-okd-4.11-upgrade-from-okd-4.10-e2e-upgrade-aws)

* [x] Fix found upgrade issues

* [x] Wait for OCP 4.11 RC to be released

* [x] [Bump FCOS version in openshift-installer](https://github.com/openshift/installer/pull/6108)
  * [x] [Cherry-pick on release-4.11](https://github.com/openshift/installer/pull/6204)

* [x] Release a new stable

* [ ] Update Assisted Installer okd-configmap to use OKD 4.11

* [ ] Reconfigure CI to use there branches, create necessary imagestreams (can be completed later)

  * [x] cluster-samples-operator
  * [x] cluster-update-keys
  * [ ] [origin](https://github.com/openshift/release/pull/30327) (optional)
  * [x] origin-branding
  * [ ] [ovn-kubernetes](https://github.com/openshift/release/pull/30329)
  * [x] [operator-marketplace](https://github.com/operator-framework/operator-marketplace/pull/481)
    * [x] Cherry-pick on release-4.11

**Comment by LorbusChris** [comment]
Updated to reflect current status
